settings.js

```diff
@@ -9,48 +9,66 @@ const mkdirp = require('mkdirp');
 
 var {remote} = require('electron');
 
-
 module.exports.dataFolder = path.join(remote.app.getPath("appData"), "Revolutionary-Games",
                                       "Launcher");
 
 module.exports.tmpDLFolder = path.join(remote.app.getPath("temp"),
                                        "Revolutionary-Games-Launcher");
 
-module.exports.installPath = path.join(module.exports.dataFolder, "Installed");
-
 module.exports.locallyCachedDLFile = path.join(module.exports.dataFolder,
                                                "saved_version_db_v2.json");
 
+module.exports.setInstallPath = (directory) => {
+    module.exports.installPath = directory;
+    console.log("Install path set to:", module.exports.installPath);
+};
+
+module.exports.getInstallPath = () => {
+    return String(module.exports.installPath);
+};
 
 // Make sure it exists. This simplifies a lot of code
 mkdirp.sync(module.exports.dataFolder);
 
 module.exports.settings = {
-    fetchNewsFromWeb: true
+    fetchNewsFromWeb: true,
+    hideLauncherOnPlay: true,
+    installDir: path.join(module.exports.dataFolder, "Installed"),
 };
 
+module.exports.insDirs = {
+    installedDir: null,
+}
+
 const settingsFile = path.join(module.exports.dataFolder, "launcher_settings.json");
+const installedDirFile = path.join(module.exports.dataFolder, "installed_directories.json");
```

Comment:
Why is this a separate file?
Wouldn't it make more sense to have this in the main settings file? The default path could be renamed to `defaultInstallDir` to free up the name for the actual current install directory.